<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDlgSvcQueryInfo</class>
 <widget class="QDialog" name="QDlgSvcQueryInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>187</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>703</width>
    <height>187</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>703</width>
    <height>187</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查看服务记录</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout1">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务项目编号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>车牌号码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>服务项目类别</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>服务子类别</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>保险金额</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtServiceID">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtPlateID">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edtInsuranceAmount">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtCategory">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtSubCategory">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>维修部门</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>维修人员</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>消费金额</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtServiceConsumptionAmount">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtServiceDepartment">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtMaintanceman">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>222</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>索赔金额</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtClaimAmount">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>结算方式</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>维修起始时间</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>维修截止时间</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>取车时间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtSettlementStyle">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtMaintanceStartTime">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtMaintanceEndTime">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edtPickupTime">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>158</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一条</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
